extensions/wikia/Chat2/js/views/views.js

```diff
@@ -109,20 +109,16 @@ var ChatView = Backbone.View.extend({
 
 		// Inline Alerts have may have i18n messages in them. If so (and they don't have 'text' yet), process the message and cache it in 'text'.
 		// This needs to be done before the template processing below so that 'text' will be set by then.
-
-		/*
 		if(this.model.get('text') == ''){
 			var params = this.model.get('msgParams'),
 				msgId = this.model.get('wfMsg');
-			if (!params || !msgId) {
+			if (!msgId) {
 				return this;
 			}
-			$().log("Found an i18n message with msg name " + msgId + " and params: " + params);
 			params.unshift(msgId);
-			var i18nText = $.msg.apply(null, params);
+			var i18nText = mw.message.apply(msgId, params).text();
```

Comment:
I'm not sure whether `mw.messages.apply` will work here - `msgId` will be passed as `this` to `mw.message`.